pytorch_lightning/profiler/xla.py

```diff
@@ -0,0 +1,106 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
```

Comment:
move this to the class?

Reply:
for the api summary, the first line here will be the text that appears in the table. See the other profilers and docs here: https://pytorch-lightning.readthedocs.io/en/latest/api_references.html#profiler-api

Reply:
Done!